contributor-ladder.md

```diff
@@ -0,0 +1,157 @@
+
+* Responsibilities include: 
+    * Continues to contribute regularly, as demonstrated by having at least [TODO: Number] [TODO: Metric] a year, as demonstrated by [TODO: contributor metrics source].
+<!-- Example: "as demonstrated by having at least 50 Github contributions per year, as shown by Devstats"-->
+
+* Requirements:
+    * Must have successful contribitons to the project, including at least one of the following:
+        * [TODO: Number] accepted PRs,
+        * Reviewed [TODO: Number] PRs,
+        * Resolved and closed [TODO: Number] Issues,
+        * Become responsible for a key project management area,
+        * Or some equivalent combination or contribution
+    * Must have been contributing for at least [TODO: Number] months
+    * Must be actively contributing to at least one project area
+    * Must have two sponsors who are also Organization Members, at least one of whom does not work for the same employer
+    * [TODO: other requirements] <!--TODO: such as repository access or enabling 2FA on their Github account -->
+
+* Privileges:
+    * May be assigned Issues and Reviews
+    * May give commands to CI/CD automation
+    * Entitled to vote in the [TODO: appropriate election]
+    * Can be added to [TODO: Repo Host] teams
+    * [TODO: Other Privileges]
+
+
+The process for a Contributor to become an Organization Member is as follows:
+
+<!-- the process of becoming an organization member is going to depend strongly on how your project manages its infrastructure. TO DO: Project leads to fil in exact details of how a contributor becomes an organization member-->
+* High skill for development and meaninful contribution to a project(s)
+* Demonstrates leadership capabilities 
+* Willing to mentor other contributors
+
+### Reviewer
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A reviewer approves pull requests before they're merged by maintainers. May only have responsibility over a subarea of the project initially, where they have demonstrated domain expertise. A reviewer must meet the responsibilites and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+    * Reviewing pull requests in a timely manner
+    * Following the reviewing guide
+* Requirements:
+    * Experience as a Contributor
+* Additional privileges:
+    * Has Github rights to approve pull requests
+
+### Maintainer
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Maintainer is a contributor with commit access. Decisions and prioritizes about the project. Maintainer roles may include Community Managers, Project Managers, Release Managers, Docs Managers, and Subproject Lead. An Maintainer must meet the responsibilites and has the requirements of an Reviewer, plus:<!-- sometimes also a project lead - if so, please state in the role defintion -->
```

Comment:
`A Maintainer is a contributor with commit access who makes decisions and about the project.`

Reply:
Resolved in this PR:  https://github.com/konveyor/community/pull/11